src/App.tsx

```diff
@@ -16,18 +16,21 @@ import Pools from '@pages/Assets/Pools/Pools';
 import Tokens from '@pages/Assets/Tokens/Tokens';
 import Discover from '@pages/Discover/Discover';
 import MyAssets from '@pages/MyAssets/MyAssets';
+import NewNftMint from '@pages/MyAssets/NftMint/NewNftMint';
+import MyPools from '@pages/MyAssets/Pools/MyPools';
 import CollectionEdit from '@pages/Nfts/Collections/CollectionEdit';
 import CollectionMint from '@pages/Nfts/Collections/CollectionMint';
 import Collections from '@pages/Nfts/Collections/Collections';
 import NftEdit from '@pages/Nfts/Nfts/NftEdit';
 import NftMint from '@pages/Nfts/Nfts/NftMint';
 import Nfts from '@pages/Nfts/Nfts/Nfts';
 import Swap from '@pages/Swap/Swap';
+import PoolCreate from '@pages/MyAssets/Pools/PoolCreate';
```

Comment:
we need to check the sorting rules for imports. I think I should go under the `'@pages/MyAssets/..`

Reply:
haven't run format, will do